lib/cloud_controller/deployment_updater/calculators/instances_to_scale_up.rb

```diff
@@ -0,0 +1,17 @@
+module VCAP::CloudController
+  module DeploymentUpdater
+    module Calculators
+      class InstancesToScaleUp
```

Comment:
**thought** Do we think theres value in these being separate classes? Vs a Scale calculator with 2 methods (scale up/ scale down)

Reply:
My starting guidelines with this refactor were to create dumb single-functions classes that only responded to `call` so I could focus on reorganizing the logic.  I think it's fine to combine these.